troposphere/static/js/models/InstanceState.js

```diff
@@ -58,6 +58,7 @@ var InstanceState = Backbone.Model.extend({
 });
 
 var get_percent_complete = function (state, activity) {
+  activity = activity || '*';
```

Comment:
Looks to me like this would leave the `error` state at "10% completed" and to 'keep polling'..

The behavior I'm looking for is that `error` is still treated as a final state, and that `error - *` where * is the tmp_status/task/activity, is still `polled` as if it were `active`